pkg/condition/condition.go

```diff
@@ -22,6 +22,27 @@ func AddOrUpdateStatusConditions(conditions []toolchainv1alpha1.Condition, newCo
 	return conditions, atLeastOneUpdated
 }
 
+// AddStatusConditions adds the given conditions *without* checking for duplicate types
+// Sets the `LastTransitionTime` to `metav1.Now()` for each given condition
+func AddStatusConditions(conditions []toolchainv1alpha1.Condition, newConditions ...toolchainv1alpha1.Condition) []toolchainv1alpha1.Condition {
+	for _, cond := range newConditions {
+		cond.LastTransitionTime = metav1.Now()
+		conditions = append(conditions, cond)
+	}
+	return conditions
+}
+
+// SetStatusConditions appends all the given conditions  without checking for duplicate types
```

Comment:
Well.. this function doesn't append anything to anything. It's simple returns a copy of the same conditions but with LastTransitionTIme set to now.
Why would you need that? Couldn't you just set the last transition time on the client side?
Or at least could we maybe call this function differently? `UpdateLastTransitionTime()` or something like this?

Reply:
let me refactor the code, I did not notice how poorly written this func is 🤦‍♂️

Reply:
on second thought, this func has so little value that I'm going to drop it and set the `LastTransitionTime` on the caller side instead.

Reply:
removed in 21363d1